mitodl/open-discussions
# Added trailing slash to API urls to avoid 301 redirects

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #2174 

#### What's this PR do?
Fixes an unnecessary 301 redirect when making API calls

#### How should this be manually tested?
Go to `/courses` and verify you no longer see 301 redirects on the APIs calls changed here
